pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -212,5 +212,12 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if errs := logs.ValidateLoggingConfiguration(&kc.Logging, field.NewPath("logging")); len(errs) > 0 {
 		allErrors = append(allErrors, errs.ToAggregate().Errors()...)
 	}
+
+	if localFeatureGate.Enabled(features.MemoryQoS) {
+		if kc.MemoryThrottlingFactor != nil && (*kc.MemoryThrottlingFactor <= 0 || *kc.MemoryThrottlingFactor > 1.0) {
+			allErrors = append(allErrors, fmt.Errorf("invalid configuration: MemoryThrottlingFactor %v must be greater than 0 and smaller than or equal to 1.0", kc.MemoryThrottlingFactor))
```

Comment:
use the API-style name `memoryThrottlingFactor` (lower-0case first char)

Reply:
s/smaller/less

Reply:
Updated.